src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -87,7 +87,7 @@ public abstract class GenerateProtoTask extends DefaultTask {
   // that may be "imported" from the source protos, but will not be compiled.
```

Comment:
nit: do we need to change the document *will not be compiled*?

Also, it looks it changed the behavior of `getIncludeDirs()`, which now returns less files, right?
I don't find `getIncludesDir()` being used anywhere tho, when and who this method will be used?

Reply:
> nit: do we need to change the document will not be compiled?

I don't understand. That's what the comment says.

> I don't find getIncludesDir() being used anywhere tho, when and who this method will be used?

It is used by Gradle itself to detect the input to the task, for task dependencies and avoiding running the task for incremental builds.

Reply:
> It is used by Gradle itself to detect the input to the task, for task dependencies and avoiding running the task for incremental builds.

I guess the important part for your question is that all the files that are no longer part of getIncludeDirs() will still be part of getSourceDirs(), so Gradle will still consider all those files to be inputs/dependencies to the task.